src/Uno.UI.Runtime.Skia.Android/Microsoft/UI/Xaml/LayoutProvider.cs

```diff
@@ -138,11 +136,37 @@ private void MeasureLayout(PopupWindow sender)
 
 		private void MeasureInsets(PopupWindow sender, WindowInsetsCompat insets)
 		{
-			//var systemInsets = insets.GetInsets(WindowInsetsCompat.Type.SystemBars())
-			//	.ToThickness()
-			//	.PhysicalToLogicalPixels();
+			var size = insets.GetInsets(WindowInsetsCompat.Type.SystemBars())
```

Comment:
@MartinZikmund  Is this relevant? I saw it was commented out so I added the implementation.

Reply:
@MartinZikmund  did you have time to check this?

Reply:
Seems this layouting logic will be relevant to https://github.com/unoplatform/uno/issues/19858, but in this issue it was happening for non-Skia target too. Still safe to include this one